archive.istio.io/build.sh

```diff
@@ -40,11 +39,12 @@ do
   git checkout $TAG
   git pull 2> /dev/null
   echo "baseurl: /$NAME" > config_override.yml
-  for f in "${FILESTOPATCH[@]}"
+
+  echo "archives:" > _data/archives.yml
+  for rel in "${TOBUILD[@]}"
```

Comment:
I see from https://github.com/istio/istio.github.io/pull/844
you want that archives.yml to be part of site generation

let's go over the list 2x not n^2
ie create archives.yml once and copy it 
(you can add it to .gitignore)